examples/astro/src/react/content-types/calendarEvent.tsx

```diff
@@ -0,0 +1,99 @@
+import type { FC } from "react";
+import type { DotCMSContentlet } from "../../types";
+import useImageSrc from "../hooks/useImageSrc";
+
+export type CalendarEventProps = DotCMSContentlet;
+
+export type LocationsAndActivities = {
+  locations: any[];
+  activities: any[];
+};
+
+const extractLocationsAndActivities = (
+  contentlet: DotCMSContentlet,
+): LocationsAndActivities => {
+  return (
+    contentlet?.reduce(
+      (acc: any, { activities, ...location }: LocationsAndActivities) => {
```

Comment:
Why use `any` if contentlet is typed?

Reply:
The `acc`, is not a contentlet, is something else and is not really important to type it, this is just an example, customer can do this component however they want